ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/QuotedIdentifier.java

```diff
@@ -27,11 +27,24 @@
  * @since 2.0.0
  */
 public class QuotedIdentifier {
-    private final String name;
+    private final int ASCII_VALUE = 128;
+    private String name;
```

Comment:
Can we keep this as a final variable?

Reply:
Updated in 3083628406bc711786e3b472071f89c46d21e09e